backend/controllers/prisonerProfile/adjudicationHistory.js

```diff
@@ -72,7 +72,7 @@ module.exports = ({ adjudicationHistoryService, elite2Api, logError, paginationS
             result.adjudicationNumber
           } </a>`,
         },
-        { text: `${result.reportDate} ${result.reportTime}` },
+        { text: `${result.reportDate} - ${result.reportTime}` },
```

Comment:
Sneaky :D

Reply:
haha. Told Dom I'd sort it in my next PR